docs/api/console.md

```diff
@@ -0,0 +1,159 @@
+# 🎮 Console
+
+An implementation of the standard `console` API.
+
+## Quick start
+
+### Log messages
+
+`Console` provides the following general purpose logging functions.
+* `console.log` for general messages.
+* `console.info` for information messages.
+* `console.warn` for warnings.
+* `console.error` for errors.
+* `console.debug` for debug level messages.
+
+Each of these has a similar interface. Each function accepts an arbitrary number of arguments. The logged message consists of a human readable representation of the arguments, separated by strings.
+
+The following two lines will both log the message "Hello World".
+```typescript
+console.log("Hello World!");
+console.log("Hello", "World!");
+```
+
```

Comment:
IMO, it would be best to show a simpler example before discussing multi-arg logging. e.g. 

```typescript
console.log("Hello from JS 👋");
```